config/secrets.yml

```diff
@@ -1,35 +0,0 @@
-# Be sure to restart your server when you modify this file.
-
-# Your secret key is used for verifying the integrity of signed cookies.
-# If you change this key, all old signed cookies will become invalid!
-
-# Make sure the secret is at least 30 characters and all random,
-# no regular words or you'll be exposed to dictionary attacks.
-# You can use `rails secret` to generate a secure secret key.
-
-# Make sure the secrets in this file are kept private
-# if you're sharing your code publicly.
-
-# Shared secrets are available across all environments.
-
-# shared:
-#   api_key: a1B2c3D4e5F6
-
-# Environmental secrets are only available for that specific environment.
-
-development:
-  secret_key_base: 352f2c2ccf5a9a77d8cc91fade3685dda45407f329c14ed0b8e4c2d6618d8475e4e4416bf20eeeec46558384ea9c9e2de97b85e20c47adb93d914576ed6b69d4
-
-test:
-  secret_key_base: a5cf81cfe4f58f3b177301038681edc63bb0ae35912ed7e6d35ba911275e53efdf05fb05d5c87095519bb09021c39e3b561bd242fe7d6a75982d8f0150cdca60
-
-# Do not keep production secrets in the unencrypted secrets file.
-# Instead, either read values from the environment.
-# Or, use `bin/rails secrets:setup` to configure encrypted secrets
-# and move the `production:` environment over there.
-
-production:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
-
-staging:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
```

Comment:
As this env variable was already defined in AWS Parameter store, we don't need to change any infra config and `Rails.application.secret_key_base` will directly pick it up.